webui/react/src/shared/styles/antd.scss

```diff
@@ -217,9 +217,12 @@ html {
     opacity: 1;
     user-select: auto;
   }
-  .ant-table .ant-table-tbody > tr > td[class*=ant-table-cell-fix-] {
```

Comment:
is there any reason why the overflow menu should have a different background color than the other cells?